hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -257,7 +257,7 @@ private void handlePlatformTransaction(
             @NonNull final ConsensusTransaction platformTxn) {
         // Get the consensus timestamp. FUTURE We want this to exactly match the consensus timestamp from the hashgraph,
         // but for compatibility with the current implementation, we adjust it as follows.
-        final Instant consensusNow = platformTxn.getConsensusTimestamp().minusNanos(1000 + 3L);
+        final Instant consensusNow = platformTxn.getConsensusTimestamp().minusNanos(1000 - 3L);
```

Comment:
Are we sure this is correct? This puts the transaction time OUTSIDE of the consTime +/- 1000 nanos range. Why would we want that?

Reply:
Actually, this fixes to put the time back in.
Subtraction (1000 + 3) means subtract 1003, which is outside.  This fixes the code to subtract 997, as originally intended.

Reply:
Ahhh I didn't see the double negative, got it 👍

Reply:
It's confusing, which is probably why it was incorrect in the first place.